api/src/functions/auth.js

```diff
@@ -116,27 +116,25 @@ export const handler = async (event, context) => {
     // `signUp()` function in the form of: `{ message: 'String here' }`.
     handler: async ({ username: email, hashedPassword, salt }) => {
       // get customerID from Stripe using email
-      const customer = await stripeCustomerSearch({
-        query: `email: \"${email}\"`,
-      })
-      let customerId = ''
-      let customerName = ''
-      if (customer !== undefined) {
-        customerId = customer.id
-        customerName = customer.name
-      } else {
-        const newCustomer = await createStripeCustomer({ email })
-        customerId = newCustomer.id
-      }
-
-      // Use Stripe details for adding new user
+      // const customer = await stripeCustomerSearch({
+      //   query: `email: \"${email}\"`,
+      // })
+      // let customerId = ''
+      // let customerName = ''
+      // if (customer !== undefined) {
+      //   customerId = customer.id
+      //   customerName = customer.name
+      // } else {
+      //   const newCustomer = await createStripeCustomer({ email })
+      //   customerId = newCustomer.id
+      // }
+
+      // Create new user
```

Comment:
Had to remove the Stripe create Customer functionality here because this seems to run before log in. I would need to query the Stripe Customer after log in.